crates/core/src/utility.rs

```diff
@@ -87,6 +87,12 @@ pub async fn download_update_to_temp_file(
         .unwrap()
         .to_str()
         .unwrap()
+        // Trim `temp_` from the `current_exe` path in case this self update process
```

Comment:
Nice documented.

Reply:
It's hard to make it make sense without way more context, but better than nothing :)